content/4_Gold/DSU.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 id: dsu
 title: 'Disjoint Set Union'
```

Comment:
is this supposed to have quotation marks?

Reply:
It looks like all modules have quotation marks for the title

Reply:
? https://usaco.guide/general/working-mdx/#example-frontmatter

Reply:
Oh yeah looks like removing quotes works too

Reply:
Just removed them, thanks!